spec/features/staff/users_can_view_transactions_on_an_activity_page_spec.rb

```diff
@@ -32,7 +32,20 @@
 
         expect(page).to have_content(transaction_presenter.financial_quarter_and_year)
         expect(page).to have_content(transaction_presenter.value)
-        expect(page).to have_content(transaction_presenter.receiving_organisation_name)
+        expect(page).to have_content(transaction.receiving_organisation_name)
```

Comment:
What's the value in changing this expectation to check for the field on the transaction rather than the result of the transaction presenter?

The new presenter method isn't tested in the presenter spec; would there be value in testing it there?

Reply:
Ah, well spotted. I've removed this tests and added a test to the presenter spec, which I agree, makes more sense. I wasn't 100% comfortable testing for the presence of `N/A` anywhere in the page anyway, because that could be introduced somewhere else